aboutUS-invest.html

```diff
@@ -0,0 +1,132 @@
+﻿<!DOCTYPE html>
+<html lang="zh-CN">
+<head>
+    <meta charset="UTF-8">
+    <title>贝贝投-网贷理财,投资理财,行业领先的互联网理财平台！</title>
+    <meta name="renderer" content="webkit">
+    <meta name="keywords" content="贝贝投官网,互联网理财,P2P理财,互联网金融,个人理财,投资理财,p2p网贷,网络借贷">
+    <meta name="description" content="贝贝投,是一家行业领先的互联网理财平台,互联网理财网贷理财平台,专注供应链金融,致力于为你实现财富稳健增值,互联网理财首选。">
+    <!-- css -->
+    <link rel="stylesheet" type="text/css" href="css/main.min.css">
+    <link rel="stylesheet" type="text/css" href="css/about.min.css">
+    <link rel="stylesheet" type="text/css" href="css/basic.css">
+    <!-- javascript -->
+    <script type="text/javascript" src="js/main.js"></script>
+</head>
+<body> 
+<!-- navbar -->
+<nav class="navbar-wrapper">
+    <div class="navbar">
+        <a href="./" class="logo-wrapper">
+            <img src="img/logo.png" alt="贝贝投" class="logo">
+        </a>
+        <ul class="nav">
+            <li><a href="#">首页</a></li>
+            <li><a href="">我要理财</a></li>
+            <li><a href="">我要借款</a></li>
+            <li><a href="">新手攻略</a></li>
+            <li class="nav-current"><a href="aboutUs-info.html">关于我们</a></li>
+        </ul>
+        <div class="nav-app left"><a href="">客户端</a></div>
+        <div class="nav-user-wrapper left"><a href="" id="nav-login">登录</a> / <a href="">注册</a></div>
+    </div>
+</nav>
+
+<!-- tab -->
+<section class="tab">
```

Comment:
This part is out of date, check out the latest version 7288e1ce9f91c7ffafa6a7fe2028d8898ff745ad.